frontend/apps/web/src/pages/system/_name.tsx

```diff
@@ -28,95 +33,224 @@ const RepositoryDetails = () => {
   const totalTests = system.subsystems.reduce((totalTests, subsystem) => {
     return totalTests + subsystem.tests.length
   }, 0)
+  const navigate = useNavigate()
+  const [activeTab, setActiveTab] = useState(1)
 
-  const pageLayout = usePageLayout({
-    header: (
+  const Header = () => {
+    return (
       <PageLayout.Header>
         <StackLayout>
           <Text type="heading 5">Systems</Text>
         </StackLayout>
+        <PageLayout.Tabs>
+          <PageLayout.Tab
+            onClick={() => setActiveTab(1)}
```

Comment:
As we talked about, use setActiveTab('overview')

and also add something like this (now we use navigate instead of history, so check how it is now):  
`useEffect(()=>{history.push({search:'overview')},[activeTab])`

Reply:
Done